### DISCUSSION TITLE
Products for processing kinematic orbits for the year 2021

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#114] [Q&A]
Dear developers, 

I have the RINEX data and SP3 orbits as well as the attitude data for a LEO constellation and I would like to process the data in the year 2021 for one month in August. Due to the fact that we need to use the TUG products to prevent inconsistencies in the processing, I am wondering what would be the alternative solution for the precise orbits, clocks and signal biases. Or, there are not any TUG products for the year 2021, are there? 

Thank you in advance for your help. 

With Kind Regards, 
Parisa

**Comment by tmayerguerr** [comment]
Dear Parisa,

The published GNSS products have been calculated uniquely for the IGS repro3. It is not planned to continue this data set at the moment.

The IGS offers corresponding orbit and clock products on an operational basis. I am not sure if the phase and code biases are also available.

regards
Torsten

**Reply by ParisaShafiei** [reply]
Dear Torsten, 

Thank you so much for your response. As you have mentioned the operational products for orbits and clocks produced by the IGS ACs, firstly, I am wondering if using these products can yield to inconsistencies in the processing or not because of the different applied algorithms and models, and secondly, since we only have the differential code biases from these ACs, I do not know how to deal with these parameters in the raw observation approach. 

On the other hand, I went through the PhD thesis of Dr. Zehenter (Kinematic orbit positioning applying the raw observation approach to observe time variable gravity) and so far I have understood that he only used the IGS products in determining the kinematic orbits of the LEO satellites, so if it's possible to use the IGS products in processing, how can we deal with the code and phase biases in processing? 

Thank you in advance for your support. 

With Kind Regards, 
Parisa

**Comment by ParisaShafiei** [comment]
Dear Torsten, 

Regarding my previous question, I am writing to you so as to ask about the possibility of using CODE products in POD. Indeed, what I am looking for is to check if there is any options implemented in the software enabling us to ignore the signal and code biases and just use the precise orbits and clocks from CODE or not. 

In fact, I am aware that using different products can lead to inconsistencies because of different algorithms, but I would like to process KIN orbits of a LEO mission in the year 2021 and I am just stuck and cannot proceed anymore. 

I wanted to check with you that if there is any possibilities to eliminate the signal biases in processing and then just use the clocks and orbit products? 

Thank you in advance for your cooperation and support.

With Kind Regards, 
Parisa
